go/store/prolly/map.go

```diff
@@ -233,6 +233,44 @@ func (m Map) IterRange(ctx context.Context, rng Range) (MapIter, error) {
 	return filteredIter{iter: iter, rng: rng}, nil
 }
 
+// GetOrdinal returns the smallest ordinal position at which the key >= |query|.
+func (m Map) GetOrdinal(ctx context.Context, query val.Tuple) (uint64, error) {
+	return m.tuples.getOrdinal(ctx, query)
+}
+
+// GetRangeCardinality returns the number of key-value tuples in |rng|.
+func (m Map) GetRangeCardinality(ctx context.Context, rng Range) (uint64, error) {
```

Comment:
Are you still using this utility? seems like you wanted a different type of range